simulator/simple_planning_simulator/src/simple_planning_simulator/simple_planning_simulator_core.cpp

```diff
@@ -563,6 +565,27 @@ void SimplePlanningSimulator::publish_acceleration()
   pub_acc_->publish(msg);
 }
 
+void SimplePlanningSimulator::publish_imu()
+{
+  sensor_msgs::msg::Imu imu;
+  imu.header.frame_id = "/base_link";
+  imu.header.stamp = now();
+  imu.linear_acceleration.x = vehicle_model_ptr_->getAx();
+  constexpr auto COV = 0.001;
+  imu.linear_acceleration_covariance.at(0) = COV;
+  imu.linear_acceleration_covariance.at(3) = COV;
+  imu.linear_acceleration_covariance.at(6) = COV;
+  imu.angular_velocity = current_odometry_.twist.twist.angular;
+  imu.angular_velocity_covariance.at(0) = COV;
+  imu.angular_velocity_covariance.at(3) = COV;
+  imu.angular_velocity_covariance.at(6) = COV;
+  imu.orientation = current_odometry_.pose.pose.orientation;
+  imu.orientation_covariance.at(0) = COV;
+  imu.orientation_covariance.at(3) = COV;
+  imu.orientation_covariance.at(6) = COV;
```

Comment:
0, 3, 6 -> 0, 4, 8?

Reply:
https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/imu_corrector/src/imu_corrector_core.cpp#L81-L92
https://github.com/autowarefoundation/autoware.universe/blob/main/common/tier4_autoware_utils/include/tier4_autoware_utils/ros/msg_covariance.hpp#L26-L34

Reply:
addressed in https://github.com/autowarefoundation/autoware.universe/pull/2843/commits/8bb8b70a55e4edaa791683ed407043c8c9c11d38